aws/sqs/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "arn" {
+  value = aws_sqs_queue.this.arn
+}
+
+output "full_queue_name" {
```

Comment:
nit, i think calling it `name` is good enough

Reply:
Fair. I thought it was a bit weird that you'd have something like:
```terraform
module "sqs" {
  queue_name = "blah"
}
```
and then later:

```terraform
queue_name = module.sqs.name
```
and those not being the same thing.